operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -211,8 +211,22 @@ type DseWorkloads struct {
 	SearchEnabled    bool `json:"searchEnabled,omitempty"`
 }
 
+// StorageConfig defines additional storage configurations
+type AdditionalVolumes struct {
+	// Mount path into cassandra container
+	MountPath string `json:"mountPath"`
+	// Name of the pvc
+	// +kubebuilder:validation:Pattern=[a-z0-9]([-a-z0-9]*[a-z0-9])?
+	Name string `json:"name"`
+	// Persistent volume claim spec
+	PVCSpec *corev1.PersistentVolumeClaimSpec `json:"pvcSpec"`
```

Comment:
Does it make sense to keep this as a pointer? Each time it is used in the code the pointer is de-referenced.

Reply:
@burmanm I did the same as what is done for field `CassandraDataVolumeClaimSpec`, but that field can be missing while pvcSpec is really required. It's no longer a pointer in my last commit